internal/backend/auth/authjwttokens/jwt.go

```diff
@@ -53,12 +53,7 @@ func New(cfg *Config) (authtokens.Tokens, error) {
 	return &tokens{signKey: key}, nil
 }
 
-func (js *tokens) Create(user sdktypes.User) (string, error) {
-	uj, err := user.MarshalJSON()
-	if err != nil {
-		return "", fmt.Errorf("marshal JSON: %w", err)
-	}
-
+func (js *tokens) Create(uid sdktypes.UserID) (string, error) {
```

Comment:
i think we shuold keep the email and userID in jwt, and we soon would need organizationID and maybe tier, 

I know you prefer minimal JWT, I do as well
but these are 4 elements that would make life much easier (maybe except for email which is not mandatory)

tier and organization_id would allow us to implement rate limit without any db access at the load balancer level

Reply:
but we need to get the user anyway from the database on each login. in that case we would have the info there?

Reply:
on login sure, but after login we don't need to fetch the user if we can, on any API call

Reply:
pass the user to remind

Reply:
now i put the entire user in the token, which is essential email and title.